feature-libs/storefinder/components/store-finder-search/store-finder-search.component.html

```diff
@@ -1,30 +1,41 @@
 <div class="container">
-  <div class="row">
+  <div class="row align-items-end">
```

Comment:
I can see classes and elements have been added/removed and the DOM structure is changed. Could we please hide these changes behind a feature toggle?